integration-cli/requirements.go

```diff
@@ -88,3 +96,14 @@ func testRequires(t *testing.T, requirements ...TestRequirement) {
 		}
 	}
 }
+
+// testRequires checks if the environment satisfies the requirements
+// for the test to run or skips the tests.
+func testRequiresWithLog(t *testing.T, msg string, requirements ...TestRequirement) {
```

Comment:
@brahmaroutu you don't see SKIP: in `make test` because SKIP is something that should be shown in a more verbose level. If you have a problem with that, probably you need to talk to golang developers as they made the design decision.

If you run the tests like `TESTFLAGS='-test.v' make test` yes you will see the clean output saying `SKIP:` and that's what the `t.Skip(r.SkipMessage)` is supposed to do with the `r.Message`. I don't think we should be reinventing the wheel here and just stick with the framework defaults (in this case pkg/testing).